lambdas/file_access_patterns/lambda_for_fileDelete/index.js

```diff
@@ -43,12 +53,52 @@ exports.deleteFile = async (event) => {
       ExpressionAttributeValues: {
         ":size": -fileData.Attributes.size,
       },
-      }).promise()
-    return response(201,"File delete success")
+    }).promise()
+  }
+  catch(err){
+    return response(err.statusCode,err.message)
+  }
+  
+  params={
+    TableName:"V-Transfer",
+    KeyConditionExpression: "#PK= :pk and begins_with(#SK,:sk)",
+    ExpressionAttributeNames:{
+        "#PK": "PK",
+        "#SK": "SK"
+    },
+    ExpressionAttributeValues:{
+        ':pk':`FILE#${reqBody.f_timestamp}`,
+        ':sk':"URL#"
+    },                        
+  }
+
+  //getting all URLs corresponding to the deleted file record
+  try{
+    var url_data = await dynamo.query(params).promise()        
+    url_data=url_data.Items;
+    if(url_data.length==0){
+      return response(201,"File delete success")
+    }
   }
   catch(err){
     return response(err.statusCode,err.message)
   }
+
+  url_data.forEach(request_item_arr_util)
+  params={
+    "RequestItems":{
```

Comment:
can we pass a direct object to `RequestItems` key, or we have to pass an `array`?

Reply:
We have to pass an array because there can be multiple items to delete.

Reply:
so it will work fine!!

Reply:
so it will work fine!!